IntoHistory/MainView/MainViewController.swift

```diff
@@ -0,0 +1,163 @@
+//
+//  MainViewController.swift
+//  IntoHistory
+//
+//  Created by Yu ahyeon on 2022/08/10.
+//
+
+import UIKit
+
+class MainViewController: UIViewController {
+
+    // MARK: - Property
+
+    private let scrollView: UIScrollView = {
+        let scrollView = UIScrollView()
+        scrollView.translatesAutoresizingMaskIntoConstraints = false
+        scrollView.showsVerticalScrollIndicator = false
+        return scrollView
+    }()
+
+    private let contentView: UIView = {
+        let view = UIView()
+        view.translatesAutoresizingMaskIntoConstraints = false
+        return view
+    }()
+
+    lazy var flagAreaImage: UIImageView = {
+        let imageView = UIImageView()
+        imageView.image = UIImage(named: "flagArea")
+        imageView.contentMode = .scaleAspectFit
+        imageView.contentMode = .center
+        return imageView
+    }()
+
+    lazy var blackboardImage: UIImageView = {
```

Comment:
p1;

접근제어자 추가 부탁드려요오오오 이 아래부터는 리뷰에선 달지 않겠습니다! 수정해주시면 감사할것같아욥🥳
```suggestion
    private lazy var blackboardImage: UIImageView = {
```

Reply:
반영했습니다.
17b65ba76c2de87fe9b7327b86c8bc5033cf8a08